webClient/src/app/app.component.ts

```diff
@@ -37,17 +38,52 @@ export class AppComponent {
               private httpService: HttpService,
               private utils: UtilsService,
               private editorControl: EditorControlService,
-              private appKeyboard: EditorKeybindingService) {
+              private appKeyboard: EditorKeybindingService,
+              @Inject(Angular2InjectionTokens.PLUGIN_DEFINITION) private pluginDefinition: ZLUX.ContainerPluginDefinition,
+              private HTTP: HttpClient) {
     this.log.debug(`Monaco object=`,monaco);
   }
 
   ngOnInit() {
+    let openWindowsStorageString : string = window.localStorage.getItem("org.zowe.editor-openWindows")
+    let activeZluxEditorsCount : number
+    let dataToSave : string
+    if(openWindowsStorageString){
+      (Date.now() - +openWindowsStorageString.split(":")[1]) < (60*60*24*1000) ? activeZluxEditorsCount = +openWindowsStorageString.split(":")[0] + 1 : activeZluxEditorsCount = 1
```

Comment:
If the timestamp in localStorage is older than 24 hours, we reset the counter here